scripts/upgradeable/transpile.sh

```diff
@@ -46,5 +42,8 @@ npx @openzeppelin/upgrade-safe-transpiler -D \
   -N 'contracts/mocks/**/*' \
   -q '@openzeppelin/'
 
+# create alias to Initializable and UUPSUpgradeable
+cp $DIRNAME/alias/*.sol contracts/proxy/utils/.
```

Comment:
I like the idea of an alias, but it seems that we'll end up removing it for 6.0, and I'd prefer not to carry technical debt. Was there a reason to not provide a "Breaking Change" entry in the changelog?

Reply:
Added this https://github.com/OpenZeppelin/openzeppelin-contracts/pull/5941/commits/4c473b5ef60607e7ec5d8cb5950faa0e970483c5

wdyt ?

Reply:
Yes, I like that! Thanks 😄